lib/circles/create.js

```diff
@@ -7,9 +7,9 @@ module.exports = function(req, res) {
         mission: req.body.mission || null,
         aim: req.body.aim || null,
         expectationsForMembers: [],
-        members: [req.user.id],
+        members: [req.user.userId],
         invited: req.body.invited || [],
-        contactPerson: req.user.id,
+        contactPerson: req.user.userId,
```

Comment:
I generally prefer clarity over brevity. 

https://signalvnoise.com/posts/3250-clarity-over-brevity-in-variable-and-method-names

Consider the case where per chance the user-object was put in a variable called `current`. In that case, we would be referring to `current.id` or `current.userId`. I consider it the clarity a decent tradeoff for the brevity. :)

You could always argue that the user object should never be called `current`, but be called `currentUser`, but this way we prevent that from ever becoming an issue.